submissions/opeyemiPeter/htmlForms/signupPage.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
Task 7 doesn't require you to create `signUp` page. What you are asked to do is to create a contact form page. Follow the instruction given. Here is a link to the task https://github.com/Dufuna-CodeCamp/Dufuna-CodeCamp20/issues/24

Reply:
how do i remove the sign up page. thanks

Reply:
@adaugochi i have tried removing the signup page.
but  do i have to wait till this is merged before pushing the signup page?